docs/manual/manual_telemetry.rst

```diff
@@ -0,0 +1,235 @@
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Supported Telemetry
+=====================
+The structure of the supported telemetry messages and how to trigger them varies between different protocols. The sections below cover these details for different protocols.
+
+Hobby Protocols
+****************
+ESC telemetry is available when using :ref:`Hobby Protocols <manual_hobby>` with Vertiq modules. Telemetry messages are sent from the module as a serial
+message at the baud rate configured on the module.
+
+The exact method for requesting telemetry varies depending on what type of hobby protocol is being used, but the structure and content of 
+the telemetry sent is the same across all hobby protocols.
+
+Structure and Content
+######################
+The structure and content of the ESC telemetry message available when using :ref:`Hobby Protocols <manual_hobby>` is based on the `telemetry
+of KISS and BLHeli ESCs <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_. 
+
+**There is one key difference between the telemetry sent by Vertiq modules and the KISS standard. The KISS standard specifies sending ERpm data in the telemetry message. ERpm data
+must be converted into RPM using the number of poles of the motor to be useful. Vertiq modules directly send RPM data, which eliminates the
+need to perform any conversion.**
+
+.. note:: If your flight controller expects to receive ERPM data from these telemetry messages, it may not interpret the RPM data sent by Vertiq modules correctly by default.
+    Additional configuration may be necessary on the flight controller to properly intepret the received RPM data as RPM and not as ERPM. Refer to your flight controller's 
+    documentation.
+
+The telemetry sent by Vertiq modules when using :ref:`Hobby Protocols <manual_hobby>` is specified in the table below. The message
+consists of 10 bytes:
+
+	+----------+-------------------------+-----------------------+
+	| **Byte** | **Content**             | **Units**             |
+	+----------+-------------------------+-----------------------+
+	| 0        | Temperature             | Deg C                 |
+	+----------+-------------------------+-----------------------+
+	| 1        | Voltage (High Byte)     | cV                    |

```

Comment:
Is the comment here asking me to write out the words centivolts instead of cV? Or is it just about the use of centivolts at all? This is the way it is and how it has been for a long time to match how KISS and BLHeli ESCs chose to send it. So not really something we can change unilaterally without a lot of work on the flight controller